HW-01/src/Name.java

```diff
@@ -0,0 +1,7 @@
+public class Name {
+    public static void main(String[] args) {
+        for (int i = 0; i < 100; i++) {
```

Comment:
you forgot fori , and text must be "my name is"

Reply:
no, I didn't forget it. Could you check again:))